.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 deploy
 assets_src/RECOVER_*
 .DS_STORE
-package-lock.json
\ No newline at end of file
+package-lock.json
```

Comment:
Sometimes it's nice not to .gitignore package-lock.json because it keeps a record of the specific version of a library in use, which can change even if package.json doesn't change.